pkgs/desktops/plasma-5/ksmoothdock/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
```

Comment:
This package isn't part of Plasma, so it should move to `pkgs/applications/misc/ksmoothdock`.

Reply:
Moved to pkgs/applications/kde/ksmoothdock

Reply:
https://github.com/NixOS/nixpkgs/pull/88204/commits/4013b56c7dc6003548eb93cfd0af10fa1895fdcb